.gitignore

```diff
@@ -53,4 +53,6 @@ fastlane/report.xml
 fastlane/README.md
 build_config
 .env.default
-app/fabric.properties
\ No newline at end of file
+akvo-flow-server-config/
```

Comment:
Does this mean we checkout the config folder within the `akvo-flow-mobile` folder?

Reply:
we clone it when deploying with travis